src/timesfm/__init__.py

```diff
@@ -11,7 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 """TimesFM init file."""
 
-from .timesfm import TimesFm, freq_map
```

Comment:
I was wondering if we should print / log the API change here, just in case people forget to check the new README.

Reply:
Oh thats a good idea. What should be log here ?

Reply:
What about

print("TimesFM v1.2.0. See https://github.com/google-research/timesfm/blob/master/README.md for updated APIs.")

Reply:
Done, will leave the version upload to you.